tasks/README.md

```diff
@@ -59,9 +59,13 @@ The preferred style of the keys should be `snake_case`.
 
 **parameters**:  The parameters or input the task accepts listed with a [Puppet data type](../language/types_values_variables.md) string and optional description. Top level params names have the same restrictions as Puppet class param names and must match `\A[a-z][a-z0-9_]*\z`. Parameters may be `sensitive` in which case the task runner should hide their values in UI where possible.
 
-**implementations**: A list of implementation files in preference order, along with an optional list of required features for that implementation to be suitable on a target. The available features are defined by the task runner, but task runners should define at least the `shell`, `powershell` and `puppet-agent` features.
+**files**: Array of file names specified as strings that include the module containing the file. Files must be saved in module directories that Puppet makes available via mount points: `files`, `lib`, `tasks`. File specifications ending with `/` will require the entire directory. Files can also be provided as `implementations` objects. 
```

Comment:
The examples below help, but it took me some back-and-forth here to understand that the `files` here are built like the `source` for puppet `file` resources. Confirming this, or explaining the "mount points" (either directly here, or by reference to pupet documentation) avoid the confusion.

Reply:
They're not really like `puppet:///` paths, they're full file paths starting from the module name.